internal/tofu/context_validate_test.go

```diff
@@ -2488,3 +2488,21 @@ locals {
 		t.Fatalf("expected deprecated warning, got: %q\n", warn)
 	}
 }
+
+// Ensure that the plantimestamp() call is not affecting the validation step.
+func TestContext2Validate_range_over_zero_plan_timestamp(t *testing.T) {
```

Comment:
nit: I am a bit confused by the naming in this file, however I believe it should be:

```suggestion
func TestContext2Validate_rangeOverZeroPlanTimestamp(t *testing.T) {
```

Reply:
https://github.com/opentofu/opentofu/commit/f46251f9d02a84e51a589a2284b6496db8fe2d9c